lexicons/app/bsky/graph/defs.json

```diff
@@ -4,9 +4,10 @@
   "defs": {
     "listViewBasic": {
       "type": "object",
-      "required": ["uri", "name", "purpose"],
+      "required": ["uri", "cid", "name", "purpose"],
       "properties": {
         "uri": {"type": "string", "format": "at-uri"},
+        "cid": {"type": "string", "format": "cid"},
```

Comment:
Just a note, we'll need to rollout the backend changes here ahead of client changes for compatibility.

Reply:
yes, aware, causing type errors on frontend so we arent gonna merge until it is released

cc:@pfrazee